Database/Updates/World/portal_destination-Jyard-4-23.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `portal_destination` (`weenieClassId`, `cell`, `x`, `y`, `z`, `qx`, `qy`, `qz`, `qw`, `min_lvl`, `max_lvl`) VALUES
+(13127, 3578855429, 18.2322, 104.646, 48.564, -0.601667, 0, 0, 0.798747, 0, 0),
+(28267, 24772919, 29.6039, -10.1276, 0.01, 1, 0, 0, 0, 10, 0),
+('28268', '3663134748', '92.442154', '76.792091', '14.004999', '-0.969302', '0', '0', '-0.245872', '0', '0');
```

Comment:
`('28268', '3663134748', '92.442154', '76.792091', '14.004999', '-0.969302', '0', '0', '-0.245872', '0', '0')`
remove the single quotes around the numbers